images/os/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM fedora:29 AS build
```

Comment:
should this be bumped up to a new version or perhaps `:latest`?

Reply:
Could be, but I am worried of breaking things that already work :D 
The other repos using similar tests still have it at 29, so just going to leave that for now.